lib/tests/streamlit/config_test.py

```diff
@@ -402,6 +453,20 @@ def test_config_option_keys(self):
                 "theme.showBorderAroundInputs",
                 "theme.linkColor",
                 "theme.showSidebarSeparator",
+                "theme.sidebar.base",
+                "theme.sidebar.primaryColor",
+                "theme.sidebar.backgroundColor",
+                "theme.sidebar.secondaryBackgroundColor",
+                "theme.sidebar.textColor",
+                "theme.sidebar.baseFontSize",
```

Comment:
I think `baseFontSize` is something that should only be supported in the main theme section.